src/actions/recomputeReduxState.js

```diff
@@ -830,3 +831,16 @@ export const createTreeTooState = ({
   // }
   return {tree, treeToo, controls};
 };
+
+function addEndOfNarrativeBlock(narrativeBlocks) {
+  const lastContentSlide = narrativeBlocks[narrativeBlocks.length-1];
+  const endOfNarrativeSlide = {
+    dataset: lastContentSlide.dataset,
+    query: lastContentSlide.query,
+    isEndOfNarrativeSlide: true
+  };
+  if (lastContentSlide.mainDisplayMarkdown) {
```

Comment:
@jameshadfield I don't know what all the possible narrative fields are that you might want to set to match (or not match) the lastContentSlide, but would it make sense to do it like this?:
```
  const endOfNarrativeSlide = Object.assign({}, lastContentSlide, {
    __html: undefined,
    isEndOfNarrativeSlide: true
  });
```

Reply:
Yes -- this is better if we ever add another field! Changed to this approach.